### PR TITLE
A11y improvements for IDP service & update embed.go

### PR DESCRIPTION
**Pull request by pascalwengerter**
**Please do not merge before we've figured this out in the web team** => need to discuss how we can deploy these with an updated a11y-branch for the re-audit

## Description
- Updated a11y feedback
- Committed `embed.go`s for `accounts` and `settings` since Dependabot PRs were merged but no binaries were ever added

## Related Issue
- Fixes https://github.com/owncloud/web/issues/5376
- Fixes https://github.com/owncloud/web/issues/5377

### CODE REVIEW

File: idp/ui/src/containers/Login/Login.js
Comment (pascalwengerter) [inline idp/ui/src/containers/Login/Login.js:109]: Think this is not 100% right, maybe a `<span>` is better than a label

**Comment by pascalwengerter** [comment]
@kulmann should we merge this for the release next week and cherrypick the commit when going for the second a11y-audit?

**Comment by micbar** [comment]
@pascalwengerter @kulmann Include in the release? then we should merge now.

**Comment by pascalwengerter** [comment]
> @pascalwengerter @kulmann Include in the release? then we should merge now.

Good to go from my side, waiting for feedback/review from Benedikt ;)

**Review by kulmann — APPROVED** [review]
LGTM 👍

**Pull request merged**
